.github/ISSUE_TEMPLATE/feature-request-issues.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Feature Request
+about: Let us know what you would like to see in the game!
+---
+
+<!-- After you fill in all information, delete all comments in the issue -->
```

Comment:
is this even valid markdown? is this used in other repo's issue templates?

Reply:
it is valid syntax for commenting